doc/user_guide/electron_holography.rst

```diff
@@ -0,0 +1,26 @@
+Electron Holography
+*******************
+
+HyperSpy provides the user with two classes which can be used to process electron holography data:
+
+* :py:class:`~._signals.hologram_image.HologramImage`
+* :py:class:`~._signals.wave_image.WaveImage`
+
+Both inherit directly from the :py:class:`~._signals.signal2d.Signal2D` class and thus can use all of its
+functionality. The usage of both classes is explained in the following sections.
+
+
+
+The WaveImage class
+===================
+
+The :py:class:`~._signals.wave_image.WaveImage` class can hold information about the complex electron

```

Comment:
In the same vein, would it be worth having a `Wave` class and move to `Wave2D` only the methods that are only for images?

The FFT (see #817) could then return a `Wave` instance.